### PR TITLE
Add Syncthing-Fork & Orgzly

### PR DESCRIPTION
**Pull request by fabian-thomas**
Orgzly is an Editor/Viewer for org files (see https://orgmode.org/).
I use it both for notes and todos, so I would propose we add it in both sections.

And I think we should also include the amazing fork of syncthing-android.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -522,6 +523,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**DecSync CC**](https://github.com/39aldo39/DecSyncCC) <sup>**[[F-Droid](https://f-droid.org/app/org.decsync.cc)]**</sup>
 * [**EteSync**](https://github.com/etesync/android) <sup>**[[F-Droid](https://f-droid.org/app/com.etesync.syncadapter)]**</sup>
 * [**Syncthing**](https://github.com/syncthing/syncthing-android) <sup>**[[F-Droid](https://f-droid.org/app/com.nutomic.syncthingandroid)]**</sup>
+* [**Syncthing-Fork**](https://github.com/Catfriend1/syncthing-android) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.github.catfriend1.syncthingandroid)]**</sup>
```
Comment (offa) [inline README.md:526]: _(see above)_
```suggestion
* [**Syncthing-Fork**](https://github.com/Catfriend1/syncthing-android) <sup>**[[F-Droid](https://f-droid.org/app/com.github.catfriend1.syncthingandroid)]**</sup>
```

File: README.md [hunk 2]
```diff
@@ -558,6 +560,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**1List**](https://github.com/lolo-io/OneList) <sup>**[[F-Droid](https://f-droid.org/app/com.lolo.io.onelist)]**</sup>
 * [**OpenTasks**](https://github.com/dmfs/opentasks) <sup>**[[F-Droid](https://f-droid.org/app/org.dmfs.tasks)]**</sup>
 * [**Tasks.org**](https://github.com/tasks/tasks) <sup>**[[F-Droid](https://f-droid.org/app/org.tasks)]**</sup>
+* [**Orgzly**](https://github.com/orgzly/orgzly-android) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.orgzly)]**</sup>
```
Comment (offa) [inline README.md:563]: Please use a single category per app only.
```suggestion
```

**Review by offa — APPROVED** [review]
Thanks! :+1: 
There are some minor editorial issues, but I'll fix these up.

**Comment by fabian-thomas** [comment]
You removed the duplicated entry of Orgzly. If we do not want it twice in the list I would propose to just include it in the Todo section since I use it more for Todos than for notes.

**Comment by offa** [comment]
If it suits better there, why not :-)

Update is on the way …

**Pull request merged**